api/Event.json

```diff
@@ -226,14 +226,24 @@
             "edge": {
               "version_added": "12"
             },
-            "firefox": {
-              "version_added": "53",
-              "notes": "Prior to Firefox 53, this property was defined on the <a href='https://developer.mozilla.org/docs/Web/API/UIEvent'><code>UIEvent</code></a> interface. See <a href='https://bugzil.la/1298970'>bug 1298970</a> for more details."
-            },
-            "firefox_android": {
-              "version_added": "53",
-              "notes": "Prior to Firefox 53, this property was defined on the <a href='https://developer.mozilla.org/docs/Web/API/UIEvent'><code>UIEvent</code></a> interface. See <a href='https://bugzil.la/1298970'>bug 1298970</a> for more details."
-            },
+            "firefox": [
+              {
+                "version_added": "53"
+              },
+              {
+                "version_added": "1",
+                "notes": "This property was defined on the <a href='https://developer.mozilla.org/docs/Web/API/UIEvent'><code>UIEvent</code></a> interface."
```

Comment:
Could we add a dedicated field to reduce this kind of prose?

Reply:
Please open an issue for this. I'm not saying no, but we need to talk to consumers about whether they would actually use such a thing (or whether we need to consider something else, like generated notes).

Reply:
Thanks, #7400

Reply:
That was duped into https://github.com/mdn/browser-compat-data/issues/3463, good old prototype chain moves. Do we need to resolve that before finishing this PR though?

Reply:
Thanks for the nudge on this. I don't think we need to resolve that first. This makes some sense as-is and I'm going to merge it.